libsolidity/parsing/Parser.cpp

```diff
@@ -204,7 +204,7 @@ ASTPointer<PragmaDirective> Parser::parsePragmaDirective()
 	nodeFactory.markEndPosition();
 	expectToken(Token::Semicolon);
 
-	if (literals.size() >= 2 && literals[0] == "solidity")
+	if (literals.size() >= 1 && literals[0] == "solidity")
```

Comment:
This is the actual bugfix. Without this the `parsePragmaVersion` was not triggered, which outputs the fatal error we depend on in the syntax checker.